packages/firefox.rb

```diff
@@ -78,8 +77,8 @@ def self.install
     FileUtils.mkdir_p "#{icon_base_path}/128x128/apps"
     FileUtils.mkdir_p "#{icon_base_path}/256x256/apps"
     FileUtils.cp_r '.', "#{CREW_DEST_PREFIX}/firefox"
-    File.write("#{CREW_DEST_PREFIX}/bin/firefox", @firefox_sh, perm: 0o755)
-    File.write("#{CREW_DEST_PREFIX}/share/applications/firefox.desktop", @firefox_desktop, perm: 0o644)
+    File.write "#{CREW_DEST_PREFIX}/bin/firefox", @firefox_sh, perm: 0o755
```

Comment:
Does FileUtils.install work for these? That way you can also get rid of a bunch of `mkdir_p`...

Reply:
Yes, changes made.